test/e2e/helpers.js

```diff
@@ -528,7 +528,7 @@ const onboardingCompleteWalletCreationWithOptOut = async (driver) => {
   await Promise.all(
     (
       await driver.findClickableElements(
-        '.toggle-button.toggle-button--on:not([data-testid="basic-functionality-toggle"] .toggle-button)',
+        '.toggle-button.toggle-button--on:not([data-testid="basic-functionality-toggle"] .toggle-button):not([data-testid="profile-sync-toggle"] .toggle-button)',
```

Comment:
In the driver actions above, we have already toggled the profile-sync button off, so we don't want to turn it back on.

NOTE - we will discuss with the team to better align with the global "Basic Functionality" toggle. (just some thoughts), but if the Basic Functionality is turned off, do we want to disable or hide our notification toggles & features. May be worth discussing with other teams on how this is solved.